docs/semgrep-code/semgrep-assistant-code.md

```diff
@@ -54,11 +54,18 @@ Semgrep Assistant supports all the same languages as Semgrep Code. See [Supporte
-* Semgrep Inc is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support). The code snippets we upload are be persisted by OpenAI temporarily, following their data usage policies at [Open AI API data Usage Policies](https://openai.com/policies/api-data-usage-policies).
-* For more details, see the [Semgrep Assistant FAQ](https://get.semgrep.dev/assistant).
+* Semgrep logs and stores the GPT prompts and responses for the sake of performance evaluation, which include source code snippets.
+* Semgrep sends relevant lines of code to OpenAI's API, where currently, the "relevant lines of code" means lines that are part of the Semgrep finding, plus 10 lines of context on each side. Semgrep is likely to expand this, potentially to the entire file, as we learn how to pass more useful context.
+* Semgrep stores and retains these code snippets for up to 6 months. Semgrep will update you with at least a 30-day notice if we make any changes to the retention policy.
+* Semgrep is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support). The code snippets we upload are be persisted by OpenAI temporarily, following their data usage policies at [Open AI API data Usage Policies](https://openai.com/policies/api-data-usage-policies).
+* * Semgrep takes several steps to protect data that is processed by AI since Assistant requires the sharing of code snippets with a third party. First, Semgrep shares code snippets with OpenAI without identifying the customer or repository name. Second, Semgrep only shares the code necessary to enlist the help of GPT in automating the resolution of each specific alert. Finally, Semgrep only accesses source code repositories on a file-by-file basis; it does not need or request org-level access to your codebase.
+* When using Semgrep Assistant, source code DOES leave your repository; Semgrep submits part of the file with a finding to OpenAI for processing by a GPT model. OpenAI is not allowed to use the submitted code for training their models.
+* Regarding your data privacy, none of your personal information is shared with OpenAI as a part of the Semgrep Assistant feature.
+* Semgrep and OpenAI do not obtain any rights to your source code. Your source code remains yours, and Semgrep or OpenAI accesses it to the limited extent necessary to provide the Semgrep Assistant service. Once the results are returned to you, Semgrep deletes the shared snippets. OpenAI retains copies of the content sent to them for a maximum of 30 days to monitor for abuse, as indicated in their API Data Usage Policies.
+* Because Semgrep will be accessing OpenAI’s services via API, OpenAI will not use any of the code provided to them to improve their services (see Section 3(c) of their Terms of Use).
```

Comment:
```suggestion
* Because Semgrep Assistant accesses OpenAI’s services through the API, OpenAI does not use any of the code provided to them to improve their services (see Section 3(c) of their Terms of Use).
```

Google Style Guide forbids "via".